src/test/kotlin/org/elm/ide/inspections/TypeInferenceInspectionTest.kt

```diff
@@ -1,1637 +0,0 @@
-package org.elm.ide.inspections
-
-class TypeInferenceInspectionTest : ElmInspectionsTestBase(ElmTypeInferenceInspection()) {
```

Comment:
I can recombine these if you'd like. IntelliJ chugs a bit when it's all a single file.